Documentation/teaching/index.rst

```diff
@@ -28,7 +28,7 @@ To get started build the documentation from the sources after
 installing docker-compose on you host:
 
 .. code-block:: c
-   
+
    cd tools/teaching && make docker-docs
```

Comment:
Since you are touching this, can you please change tools/teaching to tools/labs?

Reply:
Fixed.